ZipBuilder/Sources/ZipBuilder/main.swift

```diff
@@ -43,6 +43,7 @@ var paths = ZipBuilder.FilesystemPaths(templateDir: args.templateDir,
                                        coreDiagnosticsDir: args.coreDiagnosticsDir)
 paths.allSDKsPath = args.allSDKsPath
 paths.currentReleasePath = args.currentReleasePath
+paths.logOutputDir = args.outputDir?.appendingPathComponent("build_logs")
```

Comment:
2nd place for the string constant "build_logs"

Reply:
I was playing around and tried to make this as clean as I could by sharing a constant, but outside of adding a global `Constants` file and struct, it won't be clean.

Think it's okay to leave them as is for now? We're not depending on the names of the folders being the same for any sort of parsing - in fact the only parsing that can happen with a tool is if the `outputDir` is manually specified.

Reply:
sgtm